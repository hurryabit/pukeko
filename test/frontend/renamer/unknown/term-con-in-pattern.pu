f : a
f x =
  match x with
  | Con -> 1
