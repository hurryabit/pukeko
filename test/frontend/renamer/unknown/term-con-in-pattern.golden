test/frontend/renamer/unknown/term-con-in-pattern.pu:4:5: unknown data constructor: Con
